views/index.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Welcome to Chitter!</h1>
+<br>
+<form action="/peeps" method="post">
+  <input type="text" value="enter peep here" name="enter peep">
```

Comment:
You can use a `placeholder` attribute for 'enter peep here' which greys it out and disappears it when the user starts typing.

Reply:
Just what I was looking for! Thanks. I'll swap in ```placeholder```.

Reply:
Updated.